.travis.yml

```diff
@@ -1,14 +1,17 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.5.5
   - 2.6.0
   - 2.6.1
   - 2.6.2
   - 2.6.3
+  - 2.6.4
+  - 2.6.5
+  - 2.6.6
+  - 2.7.0
```

Comment:
Ruby 2.7.1 and 2.7.2 are out so it would be useful to add these too (and in `bin/ruby_version_test` too).

You could (optionally) add 3.0.0 as well. If we are not sure that it will pass then you can add this to the bottom of this file:

```yml
matrix:
  allow_failures:
    - rvm: 3.0.0
```

and then the tests will run with that version but will not cause the build to fail.

Reply:
👍 added in support for 2.7.1 and 2.7.2 (didn't do 3.0.0 as was having issues with that locally).